zipkin-server/src/main/java/zipkin2/server/internal/actuate/TolerantPathMapping.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright 2015-2019 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin2.server.internal.actuate;
+
+import com.linecorp.armeria.common.HttpMethod;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.server.PathMapping;
+import com.linecorp.armeria.server.PathMappingContext;
+import com.linecorp.armeria.server.PathMappingResult;
+import com.linecorp.armeria.server.VirtualHost;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import javax.annotation.Nullable;
+
+final class TolerantPathMapping implements PathMapping {
```

Comment:
I'm looking at making an armeria module for actuator, mostly by just following this code. Just wondering, why did you need this class? I'm not really following what it does, but suspect a change in core could be made to avoid it.

Reply:
took me a few minutes to remember!

yeah so we want to handle when no content type header is sent at all.

Reply:
Thanks, will check it. Wouldn't have figured that out since from what I can tell, `/loggers` is the only default endpoint that sets `consumes`!